capa/render/vverbose.py

```diff
@@ -22,8 +23,60 @@
 from capa.rules import RuleSet
 from capa.engine import MatchResults
 
+logger = logging.getLogger(__name__)
 
-def render_locations(ostream, locations: Iterable[frz.Address]):
+
+def _get_process_name(layout: rd.DynamicLayout, addr: frz.Address) -> str:
+    for p in layout.processes:
+        if p.address == addr:
+            return p.name
+    logger.debug("name not found for process: %s", addr)
+    return ""
+
+
+def _get_call_name(layout: rd.DynamicLayout, addr: frz.Address) -> str:
+    call = addr.to_capa()
+    assert isinstance(call, capa.features.address.DynamicCallAddress)
+
+    thread = frz.Address.from_capa(call.thread)
+    process = frz.Address.from_capa(call.thread.process)
+
+    # danger: O(n**3)
+    for p in layout.processes:
+        if p.address == process:
+            for t in p.matched_threads:
+                if t.address == thread:
+                    for c in t.matched_calls:
+                        if c.address == addr:
+                            return c.name
+    logger.debug("name not found for call: %s", addr)
+    return ""
+
+
+def render_process(layout: rd.DynamicLayout, addr: frz.Address) -> str:
+    process = addr.to_capa()
+    assert isinstance(process, capa.features.address.ProcessAddress)
+    name = _get_process_name(layout, addr)
+    return f"{name}[{process.pid}]"
```

Comment:
```suggestion
    return f"{name}[PID:{process.pid}]"
```

Reply:
to be clearer in the output?

Reply:
although it may conflict with the below `:` formatting...

Reply:
is it also helpful to include the parent process or maybe show a process tree layout at the top?

Reply:
we collect the ppid so we can be sure there's not a pid-reuse issue; however, across all the Avast reports there weren't any pid collisions (maybe CAPE doesn't handle this well??). so, i'm not sure its worth showing the ppid here to the user, since it almost certainly won't be useful.

Reply:
agree that maybe we could be more obvious about what these numbers are, perhaps like:

`[pid: 123, tid: 4558]`

Reply:
![image](https://github.com/mandiant/capa/assets/156560/52c922df-14b5-4b89-8440-a97249b3bb3c)

re-introduced the call index so that users understand why the same API call may be rendered multiple times.